<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>754</width>
    <height>335</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="Pumps">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>System</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QGraphicsView" name="graphicsView"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>-1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>+1</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Pumps</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0" colspan="2">
         <widget class="QLabel" name="label_8">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>Remaining Volume</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLabel" name="label_6">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>Flow Rate</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="4" colspan="2">
         <widget class="QLabel" name="label_7">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>      Volume to Dispense</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Pump 1</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QProgressBar" name="rem_vol_1">
          <property name="mouseTracking">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string/>
          </property>
          <property name="toolTipDuration">
           <number>-1</number>
          </property>
          <property name="whatsThis">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Remaining Volume in pump&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="value">
           <number>24</number>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="start_stop_1">
          <property name="text">
           <string>Start/Stop</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QLineEdit" name="flow_rate_1"/>
        </item>
        <item row="1" column="4">
         <widget class="QLineEdit" name="vol_to_dispense_1"/>
        </item>
        <item row="1" column="5">
         <widget class="QPushButton" name="set_1">
          <property name="text">
           <string>Set</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QLabel" name="label_12">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>Remaining Volume</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QLabel" name="label_10">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>Flow Rate</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="4" colspan="2">
         <widget class="QLabel" name="label_11">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>     Volume to Dispense</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Pump 2</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QProgressBar" name="rem_vol_2">
          <property name="value">
           <number>24</number>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QPushButton" name="start_stop_2">
          <property name="text">
           <string>Start/Stop</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QLineEdit" name="flow_rate_2"/>
        </item>
        <item row="3" column="4">
         <widget class="QLineEdit" name="vol_to_dispense_2"/>
        </item>
        <item row="3" column="5">
         <widget class="QPushButton" name="set_2">
          <property name="text">
           <string>Set</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0" colspan="2">
         <widget class="QLabel" name="label_2">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>Remaining Volume</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="4" column="3">
         <widget class="QLabel" name="label_3">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>Flow Rate</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="4" column="4" colspan="2">
         <widget class="QLabel" name="label_4">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>    Volume to Dispense</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Pump 3</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QProgressBar" name="rem_vol_3">
          <property name="value">
           <number>24</number>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QPushButton" name="start_stop_3">
          <property name="text">
           <string>Start/Stop</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="5" column="3">
         <widget class="QLineEdit" name="flow_rate_3"/>
        </item>
        <item row="5" column="4">
         <widget class="QLineEdit" name="vol_to_dispense_3"/>
        </item>
        <item row="5" column="5">
         <widget class="QPushButton" name="set_3">
          <property name="text">
           <string>Set</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0" colspan="2">
         <widget class="QLabel" name="label_16">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>Remaining Volume</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="6" column="3">
         <widget class="QLabel" name="label_14">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>Flow Rate</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="6" column="4" colspan="2">
         <widget class="QLabel" name="label_15">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>   Volume to Dispense</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>Pump 4</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QProgressBar" name="rem_vol_4">
          <property name="value">
           <number>24</number>
          </property>
         </widget>
        </item>
        <item row="7" column="2">
         <widget class="QPushButton" name="start_stop_4">
          <property name="text">
           <string>Start/Stop</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="7" column="3">
         <widget class="QLineEdit" name="flow_rate_4"/>
        </item>
        <item row="7" column="4">
         <widget class="QLineEdit" name="vol_to_dispense_4"/>
        </item>
        <item row="7" column="5">
         <widget class="QPushButton" name="set_4">
          <property name="text">
           <string>Set</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Injector</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Heater</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Page</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>754</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionClose"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionClose">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
